lib/kubernetes-deploy/ejson_secret_provisioner.rb

```diff
@@ -17,75 +17,41 @@ class EjsonSecretProvisioner
     EJSON_SECRETS_FILE = "secrets.ejson"
     EJSON_KEYS_SECRET = "ejson-keys"
```

Comment:
nit: I think it's ok to keep the "management annotation" on the resources, because it might be useful for people to be able to see the secret was created that way. However, I think we should rename the `MANAGEMENT_ANNOTATION` constant because we're no longer using it for that purpose.